libs/net/kafka/KafkaListener`.cs

```diff
@@ -19,7 +19,7 @@ public class KafkaListener<TKey, TValue> : IKafkaListener<TKey, TValue>, IDispos
     private readonly ILogger _logger;
     private bool _disposed = false;
     private bool _open = false;
-    private ConsumeResult<TKey, TValue>? _currentResult = null;
+    private Task? _activeHandler;
```

Comment:
Significant changes to this class and logic.  It now can correctly handle long running jobs.